ragna/source_storages/_chroma.py

```diff
@@ -32,59 +32,62 @@ def __init__(self) -> None:
                 anonymized_telemetry=False,
             )
         )
+        self._tokens = 0
+        self._embeddings = 0
 
     def store(
         self,
-        documents: list[Document],
+        documents: list[Embedding],
```

Comment:
This should be 

```suggestion
        embeddings: list[Embedding],
```

now

Reply:
There are some issues with doing this since they both stem from the same class which specifies a pydantic BaseModel.  calling them different names causes AttributeErrors.  Perhaps they could be called `data` or something more generic?  Forgive me if there is a simpler way of doing this, I am not very familiar with pydantic.

Reply:
I'm confused. Could you please post the error? Not sure how Pydantic is involved here. The name of the variable should be arbitrary.